can-bus/json/INVL/INVL_alerts.json

```
@@ -1,4 +1,3 @@
 {
-    "msg_id": 98,
-    "alerts": []
+    
 }
```

nit: missing blank line
```suggestion
}

```